tools/tests/test_lint.py

```
@@ -4472,3 +4473,251 @@ def test_hollow_code_span_caps_an_opening_fence_at_three_spaces(tmp_path):
+     + '    first, _, _ = text.partition(chr(10) + "---" + chr(10))' + NL
+     + "    return first" + NL, False),
+    ("a marker mentioned but never sliced",
+     "def check(text, rows):" + NL
+     + '    if text.startswith("---") and not text:' + NL
+     + '        raise ValueError("x")' + NL
+     + "    return rows[0]" + NL, False),
+]
+
+
+@pytest.mark.parametrize(
+    "label,source,caught",
```

Ruff (0.16.2)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)